src/main/java/frc/robot/bindings/KeyboardBindings.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.bindings;
+
+import edu.wpi.first.wpilibj2.command.PrintCommand;
+import frc.utils.controllers.keyboard.KeyboardController;
+
+public class KeyboardBindings {
+
+    private static KeyboardController KEYBOARD_CONTROLLER;
+
+    public static void configureBindings() {
+        KEYBOARD_CONTROLLER = new KeyboardController();
```

Comment:
are you sure it doesn't create like that?
also if you can put it in the KEYBOARD_CONTROLLER varibale itself without it to create it it would be better.